### ISSUE TITLE
Update _build_base_phdc to include and empty structuredBody element

### ISSUE THREAD

**Issue by DanPaseltiner** [CDCgov/phdi#1142]
When we produce an empty PHDC during initialization of `PHDCbuilder` we do not currently created an empty body. We will need this so that the soon to be implemented `build_clinical_information` method has a place to put the data it will produce.

GIVEN the `PHDCbuilder` class
WHEN the `_build_base_phdc()` method is called 
THEN  a ET.ElementTree object representing the following XML is returned.

```
<?xml version='1.0' encoding='utf-8'?>
<?xml-stylesheet type="text/xsl" href="PHDC.xsl"?>
<ClinicalDocument xmlns="urn:hl7-org:v3" xmlns:sdt="urn:hl7-org:sdtc" xmlns:sdtcxmlnamespaceholder="urn:hl7-org:v3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:hl7-org:v3 CDA_SDTC.xsd">
	<component>
		<structuredBody>
		</structuredBody>
	</component>
</ClinicalDocument>

```

**Comment by DanPaseltiner** [comment]
This may require tweaking the `builder_header()` method it ensure it inserts header data in the correct location. Failing tests should indicate is this becomes an issue.

**Comment by robertmitchellv** [comment]
i implemented this and worked through some of the tests locally that were failing and it would've required more tweaking since the `<ClinicalDocument>` is the root and when we append to the root in `build_header` it places the elements _after_ the `<component>` and `<structuredBody>` tags from `_build_base_phdc`, which doesn't follow the structure from our example PHDC documents. 

it makes sense to me to have the `_build_base_phdc` simply create the root and then have the `build_header` and `build_body` create those sections as factories to ensure that the methods are being applied to the correct sections. i was in the middle of doing this when i looked more closely at Zedd's PR while helping with diagnosing the issue with the pre-commit bot and the linting workflow. i'm not sure what your thoughts are on this; maybe you disagree completely, but an alternative would be to let Zedd's PR merge to main and then work on either: 
* retrofitting the `<component>` and `<structuredBody>` into `_build_base_phdc`; or,
* a combination of that _plus_ picking the Director back up

let me know what you think

**Comment by robertmitchellv** [comment]
Solved in #1163
